packages/pages/src/util/editConfigYaml.ts

```diff
@@ -33,4 +41,12 @@ export const addResponseHeadersToConfigYaml = (
   }
 
   fs.writeFileSync(configYamlPath, YAML.stringify(yamlDoc));
+
+  if (comment) {
+    const yaml = YAML.parseDocument(fs.readFileSync(configYamlPath, "utf-8"));
+    const responseHeader: any = yaml.get("responseHeaders");
+    responseHeader.comment = comment;
```

Comment:
Doesn't this add the comment to the top of the responseHeaders? If you have multiple modules it's going to have multiple comments in a row saying the same thing. I think we want it above each new entry we auto add. Is it possible to add nodes to the yaml doc so you can add comments at the same time or can you only add via json above?

Reply:
This just adds one comment! 

I tried to add it via nodes but it got messier than I'd prefer and required weird formatting due to the fact that responseHeaders is an array of objects. Therefore in the end I decided keeping the simple JSON method was cleaner + had less room for error.

Reply:
The issue is that there could be other responseHeaders the user adds.

Also, the text is "This response header allows access to your modules from other sites". Let's verify with Aaron that we actually want this at all. If so then we should change the text.